lib/bolt/transport/local.rb

```diff
@@ -32,8 +32,6 @@ def in_tmpdir(base)
             yield dir
           end
         end
-      rescue StandardError => e
-        raise Bolt::Node::FileError.new("Could not make tempdir: #{e.message}", 'TEMPDIR_ERROR')
```

Comment:
I guess I was trying to be helpful, but it didn't work very well.

Reply:
A test probably needs updated for this. You can run the local transport specific tests via `rake integration:bash`.

Reply:
Yeah, the problem is that the block form of this method requires makes it hard to rescue *only* tmpdir-related errors... We could do something like wrap the `yield` in a begin/rescue that specially wraps any exception it catches and rethrows, then have the outer begin/rescue rethrow the inner exception. But that seems a little over the top, considering tmpdir errors are unlikely to happen in the first place.